content/docs/guides/_index.md

```diff
@@ -21,3 +21,7 @@ weight: 30
 
 - [Receive and Send Messages Using Waku Relay With ReactJS](./07_reactjs_relay/)
 - [Retrieve Messages Using Waku Store With ReactJS](./08_reactjs_store/)
+
+## Service Nodes
+
+- [Nwaku Service Node](./10_nwaku/)
```

Comment:
fyi, not sure if it affects anything but the PR I just merged with the angular docs is also `10`.